tests/contrib/grpc/test_grpc.py

```diff
@@ -149,6 +157,34 @@ def test_pin_can_be_defined_per_channel(self):
         channel1.close()
         channel2.close()
 
+class GrpcServerTestCase(GrpcServerBaseMixin, unittest.TestCase):
```

Comment:
We also should have tests that the appropriate `grpc.request` spans are being created for each request.